modules/nw-metallb-example-addresspool.adoc

```diff
@@ -62,3 +62,40 @@ spec:
   - 10.0.100.0/28
   - 2002:2:2::1-2002:2:2::100
 ----
+
+== Example: Assign address pools to services or namespaces
+You can assign IP addresses from an `IPAddressPool` to services and namespaces that you specify.
+
+If you assign a service or namespace to more than one address pool, MetalLB uses an available IP address from the higher priority address pool. If no IP addresses are available from the assigned address pools with a priority set, MetalLB uses an IP address from an address pool with no priority. 
+
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: doc-example-service-allocation
+  namespace: metallb-system
+spec:
+  addresses:
+    - 192.168.20.0/24
+  serviceAllocation: <1>
+    priority: 50 <2>
+    namespaces:
+      - namespace-a <3>
+      - namespace-b
+    namespaceSelectors:
+      - matchLabels:
+          zone: east <4>
+    serviceSelectors:
+      - matchExpressions: 
+      - key: security <5>
```

Comment:
intendation is not correct here. you may refer to this [example](https://github.com/metallb/metallb/pull/1821/files#diff-e12b1d6f1ff1fe562e260b97d0e1d9580c011d61d061ef402053ae0611fafec6).